### ISSUE TITLE
[Bug] new release  v0.18.1 broke relationships

### ISSUE THREAD

**Issue by yeomann** [doug-martin/nestjs-query#493]
Hi,

Thanks for the great library, we using Nx based multiple apps that extends some entity from core module in multiple Nestjs projects.

we had defined the relationships this way for example in user entity
```
  @ManyToOne(() => Agent)
  @JoinColumn({ name: 'agent_id' })
  agent?: Agent;
```
After update to v0.18.1 graphql was broken on accessing the relationships and give the entitymetadata error.
![image](https://user-images.githubusercontent.com/5970751/91710429-e9484d80-eb8c-11ea-87a4-d8b628234d52.png)

we found the fix which is to write the second argument in like this ```agent => agent.user```
due to complexity of the app and sharing of entities from core to multiple nestjs projects, it not possible to pass the second argument in some cases, that is why we had to downgrade to 0.17.10

The above explained scenario was working until the update. It's seems a big bug to be honest.
Kindly can you please check it?
Thanks

**Comment by doug-martin** [comment]
@yeomann sorry about that, all my tests assumed the relationships were bi-directional. I'm working on a fix with tests to cover this use case in the future.

**Comment by yeomann** [comment]
Thank you @doug-martin waiting for the fix, appreciate your effort.

**Comment by JeffSchofield** [comment]
I'm having this issue as well with uni-directional one-to-one relationships. I started working on a PR but have had only trouble getting tests to run locally.

I did a quick fix by changing line 224 in [`/packages/query-typeorm/src/query/relation-query.builder.ts`](https://github.com/doug-martin/nestjs-query/blob/master/packages/query-typeorm/src/query/relation-query.builder.ts#L224) to:

```ts
    const inverseEntityMetadata = relation.inverseRelation?.entityMetadata ?? relation.inverseEntityMetadata;
    const fromPrimaryKeys = inverseEntityMetadata.primaryColumns.map((pk) => ({
```

and that got my project working. This could possibly be slimmed down to:

```ts
    const fromPrimaryKeys = relation.inverseEntityMetadata.primaryColumns.map((pk) => ({
```

as I am pretty sure `relation.inverseEntityMetadata` will be available for both uni-direction and bi-directional relations but I have not been able to write a test case.
